chapter16/server/server.js

```diff
@@ -1,7 +1,7 @@
 var express = require('express');
 var bodyParser = require('body-parser');
 var AWS = require('aws-sdk');
-var uuidv4 = require('uuid/v4');
+var { v4: uuidv4 }  = require('uuid');
 var multiparty = require('multiparty');
 
 var lib = require('./lib.js');
```

Comment:
regions below: is setting the region still needed on ec2? i might be wrong but it could work without it (via IMDS).